### ISSUE TITLE
How does is work?

### ISSUE THREAD

**Issue by Tibsi** [ndbiaw/jewcob-fakku-downloader#2]
I'm not really sure how this is supposed to work.
I installed step by step have a file urls.txt and tried some links.
But every time I execute main.py it just says nothing to rip. 
I tried it with the -I and -p and my credentials but no difference. 

Is it supposed to open a Chrome window or does that happen in the background. 

![Screenshot 2023-10-08 120419](https://github.com/ndbiaw/jewcob-fakku-downloader/assets/45721845/ba1fb8e5-eb0c-4c77-a946-c7f260eddf37)

**Comment by Tibsi** [comment]
OK apparently It could not read my default character encoding so I made a new text file. 
But nor I get the following:

E:\Programs installed\jewcob-fakku-downloader-main>python main.py
Cookies file(cookies.json) are not detected. Please, login in next step for generate cookie for next runs.
Initializing browser
Connecting to chromedriver
Traceback (most recent call last):
  File "E:\Programs installed\jewcob-fakku-downloader-main\main.py", line 215, in <module>
    main()
  File "E:\Programs installed\jewcob-fakku-downloader-main\main.py", line 206, in main
    loader.init_browser(auth=True, gui=args.gui)
  File "E:\Programs installed\jewcob-fakku-downloader-main\downloader.py", line 480, in init_browser
    self.__auth()
  File "E:\Programs installed\jewcob-fakku-downloader-main\downloader.py", line 644, in __auth
    self.browser = uc.Chrome(
                   ^^^^^^^^^^
  File "C:\Users\tibsi\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\__init__.py", line 373, in __init__
    if headless or options.headless:
                   ^^^^^^^^^^^^^^^^
AttributeError: 'ChromeOptions' object has no attribute 'headless'

**Comment by nayumiDEV** [comment]
Have you try Chrome v110?

**Comment by Tibsi** [comment]
How would I even do that?
The only older versions of Chrome that I can find are up to v104.

I eventually realized that installing selenium-interceptor==1.0.2 auto installs selenium 2.12 or newer which supposedly doesn't support headless mode anymore. So I reinstalled an older version of selenium, but then it gave me an error that it is only compatible with Chrome v114 or later.  Then I updated undetected_chromedriver to the newest version and now it would launch the browser and let me type in my credentials but then gives me the following error:

--------------
E:\Programs installed\jewcob-fakku-downloader-main>python main.py
Using cookies file: cookies.json
Initializing browser
Connecting to chromedriver
Traceback (most recent call last):
  File "E:\Programs installed\jewcob-fakku-downloader-main\main.py", line 215, in <module>
    main()
  File "E:\Programs installed\jewcob-fakku-downloader-main\main.py", line 209, in main
    loader.init_browser(gui=args.gui)
  File "E:\Programs installed\jewcob-fakku-downloader-main\downloader.py", line 541, in init_browser
    self.__set_cookies()
  File "E:\Programs installed\jewcob-fakku-downloader-main\downloader.py", line 606, in __set_cookies
    cookie["expiry"] = int(cookie["expiry"])
                           ~~~~~~^^^^^^^^^^
KeyError: 'expiry'
--------------------

So it somehow doesn't like cookies.json I assume? 
But I don't really know how to fix this now...
